spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -202,6 +205,17 @@ public Port setProtocol(SessionProtocol protocol) {
             protocols = ImmutableList.of(protocol);
             return this;
         }
+
+        @Override
+        public String toString() {
+            return "Port{" +
+                   "ip='" + ip + '\'' +
+                   ", address=" + address +
+                   ", iface='" + iface + '\'' +
```

Comment:
question: It seems like we add `''` to only `ip` and `iface`. Is there any specific reason for that?

Reply:
I used Intellij to generate and looks like it adds `'` or not by String or not.
Maybe I should use `MoreObjects.toStringHelper` like others?

Reply:
Didn't know about it. 😄 It looks good now. Thanks!